packages/website-eslint/build.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-process-exit, no-console */
+
+import * as fs from 'node:fs/promises';
+import { createRequire } from 'node:module';
+import * as path from 'node:path';
+
+import * as esbuild from 'esbuild';
+
+function requireResolved(targetPath: string): string {
+  return createRequire(__filename).resolve(targetPath);
+}
+
+function normalizePath(filePath: string): string {
+  return filePath.replace(/\\/g, '/');
+}
+
+function requireMock(targetPath: string): Promise<string> {
+  return fs.readFile(requireResolved(targetPath), 'utf8');
+}
+
+function makeFilter(filePath: string | string[]): { filter: RegExp } {
+  const paths = Array.isArray(filePath) ? filePath : [filePath];
+  const norm = paths.map(item =>
+    normalizePath(item).replace(/\//g, '[\\\\/]').replace(/\./g, '\\.'),
+  );
+  return { filter: new RegExp('(' + norm.join('|') + ')$') };
+}
+
+async function buildPackage(name: string, file: string): Promise<void> {
+  const eslintRoot = requireResolved('eslint/package.json');
+  const linterPath = path.join(eslintRoot, '../lib/linter/linter.js');
+  const rulesPath = path.join(eslintRoot, '../lib/rules/index.js');
+
+  const output = await esbuild.build({
+    entryPoints: {
+      [name]: requireResolved(file),
+    },
+    format: 'cjs',
+    platform: 'browser',
+    bundle: true,
+    external: [],
+    minify: true,
+    treeShaking: true,
+    write: true,
+    target: 'es2020',
+    sourcemap: 'linked',
+    outdir: './dist/',
+    supported: {},
+    define: {
+      'process.env.NODE_ENV': '"production"',
+      'process.env.NODE_DEBUG': 'false',
+      'process.env.IGNORE_TEST_WIN32': 'true',
+      'process.env.DEBUG': 'false',
+      'process.emitWarning': 'console.warn',
+      'process.platform': '"browser"',
+      'process.env.TIMING': 'undefined',
+      'define.amd': 'false',
```

Comment:
esbuild sadly don't support amd and its not able to process it, this line creates short circut of all amd wrapped packages (browser version)